generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -55,89 +54,73 @@ import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
+import static com.tngtech.sstestsupport.web.rest.AccountResourceIT.TEST_USER_LOGIN;
 import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
 /**
  * Integration tests for the {@link AccountResource} REST controller.
  */
-@SpringBootTest(classes = {<%= mainClass %>.class, TestSecurityConfiguration.class})
+@WithMockUser(value = TEST_USER_LOGIN)
+@SpringBootTest(classes = {<%= mainClass %>.class, TestSecurityConfiguration.class, NoOpMailConfiguration.class})
 <%_ if (cacheProvider === 'redis') { _%>
 @ExtendWith(RedisTestContainerExtension.class)
 <%_ } _%>
 public class AccountResourceIT <% if (databaseType === 'cassandra') { %>extends AbstractCassandraTest <% } %>{
-
-    @Autowired
-    private ExceptionTranslator exceptionTranslator;
+    static final String TEST_USER_LOGIN = "test";
 
     @Autowired
     private UserService userService;
 
-    private MockMvc restUserMockMvc;
+    private MockMvc restAccountMockMvc;
 
     @Autowired
     private WebApplicationContext context;
 
-    @BeforeEach
-    public void setup() {
-        MockitoAnnotations.initMocks(this);
-        AccountResource accountUserMockResource = new AccountResource(userService);
-
-        this.restUserMockMvc = MockMvcBuilders.standaloneSetup(accountUserMockResource)
-            .setControllerAdvice(exceptionTranslator)
-            .build();
-    }
-
     @Test
     public void testNonAuthenticatedUser() throws Exception {
-        restUserMockMvc.perform(get("/api/authenticate")
+        restAccountMockMvc.perform(get("/api/authenticate")
             .accept(MediaType.APPLICATION_JSON))
             .andExpect(status().isOk())
             .andExpect(content().string(""));
     }
 
     @Test
     public void testAuthenticatedUser() throws Exception {
-        restUserMockMvc.perform(get("/api/authenticate")
+        restAccountMockMvc.perform(get("/api/authenticate")
             .with(request -> {
-                request.setRemoteUser("test");
+                request.setRemoteUser("someUser");
```

Comment:
Maybe create also a constant?

Reply:
I usually don't create constants for things I only use in a very local context. Or do you mean a variable within the test method?

Reply:
Yes, I mean a variable

Reply:
Done :+1: